src/blosc2/proxy.py

```diff
@@ -129,6 +129,25 @@ def fetch(self, item=None):
         -------
         out: :ref:`NDArray` or :ref:`SChunk`
             The local container used to cache the already requested data.
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> import blosc2
+        >>> data = np.arange(20).reshape(10, 2)
+        >>> ndarray = blosc2.asarray(data)
+        >>> proxy = blosc2.Proxy(ndarray)
+        >>> full_data = proxy.fetch()
+        >>> f"Full data cache: {full_data[:]}"
```

Comment:
I think this would be more graphical if you fetch just an slice (say `[slice(0, 3), slice(0,2)`, so that when data is printed, one can see that only that slice has been fetched (i.e. other values are uninitialized).